packages/cli/src/ceramic-daemon.ts

```diff
@@ -208,9 +208,8 @@ class CeramicDaemon {
    * Get document state
    */
   async state (req: Request, res: Response, next: NextFunction): Promise<void> {
-    const docId = DocID.fromString(req.params.docid, req.query.version ? req.query.version.toString() : undefined)
     try {
-      const doc = await this.ceramic.loadDocument(docId)
+      const doc = await this.ceramic.loadDocument(req.params.docid)
```

Comment:
is there a reason why you removed `DocID.fromString(req.params.docid)`? If we do it like this we're going to pass a `string` to `loadDocument`.

Reply:
also, we should definitely wrap `const docId = DocID.fromString(req.params.docid)` which is used in many methods here since it can throw an error. We should call `next(err)` then. Maybe we can do that as part of another story.

Reply:
👍